Main menu page and content card

AndreLaranjeira opened the pull request:
<!--- Use o template abaixo para as Pull Requests -->

## Descrição
<!--- Faça uma lista sucinta das modificações propostas -->
* Created component for content card;
* Created main menu page;
* Added package-lock.json;
* Updated yarn.lock and android package list;

## Motivation and Context
<!--- Porque essas mudanças foram implementadas? -->
<!--- Caso essa PR encerre uma issue, apenas cite a issue usando o "Closes #123", caso contrário, explique o porque das mudanças -->
Closes #4 and closes #7.

## Como as mudanças foram testadas?
<!--- Descreva os testes realizados. -->
<!--- Para o caso dos testes automáticos, descreva brevemente quais cenários foram testados -->
<!--- Descreva brevemente quais modificações foram feitas no conjunto de testes automáticos, por exemplo, -->
<!--- quais novas Models foram migradas para o banco de testes, se foi necessário modificar alguns teste previamente existente, etc. -->
No meu celular.

## Screenshots (se aplicável):

## Tipo de modificação
<!--- É possível marcar o checklist após a PR aberta -->
- [ ] Bug fix (modificações que corrigem um bug)
- [x] Feature (modificações que adicionam uma nova funcionalidade)
- [ ] Breaking change (fix ou feature que faz com que funcionalidade(s) já existente(s) não funcione(m) como esperado)

## Checklist:
<!--- É possível marcar o checklist após a PR aberta -->
- [x] Código segue os padrões de estilo do projeto, ou seja, ESLint executa sem erros ou warnings.
- [ ] A documentação com o Swagger foi atualizada com as modificações.
- [ ] Foram adicionados testes do Jest que cobrem essa modificação.

AndreLaranjeira committed:
Main menu page and content card
* Created component for content card;
* Created main menu page;
* Added package-lock.json;
* Updated yarn.lock and android package list;

AndreLaranjeira committed:
Reconfigured eslinter parser and changes to version management
* Switched eslintrc config to use typescript-eslint/parser as parser;
* Downgraded typescript-eslint packages to avoid unfixable "react used 
before import" errors;
* Removed yarn.lock;

AndreLaranjeira committed:
Adjusted main menu page to use page layout and prop fixes
* Main menu page now uses main layout;
* Changed content card props to be optional;
* Added default props to layout;

Hugo-NF commented on src/components/ContentCard/styles.ts:
```diff
@@ -0,0 +1,23 @@
+export const styledComponents = {
+
+  CardContainer: styled.View<ViewProps>`
+    backgroundColor: #fff;
```
A boa e velha confusão entre RN.Stylesheets e StyledComponents

Hugo-NF committed:
🔥 Fix styling mistakes and removes placeholder

- Removes header placeholder from main menu page
because it's likely that it will be part of layout

Hugo-NF reviewed (approved):
Flawless implementation, as always
